extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -325,7 +325,9 @@ export async function defaultAzureAccountServiceFactory(): Promise<vscodeMssql.I
 export async function getDefaultPublishDeploymentOptions(project: ISqlProject): Promise<mssql.DeploymentOptions | vscodeMssql.DeploymentOptions> {
 	const schemaCompareService = await getSchemaCompareService();
 	const result = await schemaCompareService.schemaCompareGetDefaultOptions();
-	const deploymentOptions = result.defaultDeploymentOptions;
+	let deploymentOptions = result.defaultDeploymentOptions;
+	// optionsMapTable and includeObjects options are lowercase and the first charater of all properties should needs to be converted to uppercase.
+	deploymentOptions.optionsMapTable = convertKeysToUpperCase<mssql.DacDeployOptionPropertyBoolean>(deploymentOptions.optionsMapTable);
```

Comment:
Why do you need to convert the keys to upper case? In general everything on the extension side should be camelCased and everything on the C# side should be PascalCased - and the JSON-RPC layer should take care of converting that for you.

Reply:
The keys probably shouldn't be used as display values - since those shouldn't be localized (I believe this is called out in other comments). Have you thought about adding a displayName field or something instead that would contain the localized text to display in the UI?